it/spring/boot2-tomcat9/src/test/java/com/linecorp/armeria/spring/tomcat/demo/ErrorHandlingTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.spring.tomcat.demo;
+
+import static net.javacrumbs.jsonunit.fluent.JsonFluentAssert.assertThatJson;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.linecorp.armeria.spring.LocalArmeriaPort;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+public class ErrorHandlingTest {
+
+    @LocalArmeriaPort
+    private int port;
+
+    @Inject
+    private TestRestTemplate restTemplate;
+
+    private static String baseUrl(int port) {
+        return "http://localhost:" + port + "/tomcat/api/rest/v1";
```

Comment:
So where `/tomcat/api/rest/v1` is from? 😄

Reply:
See https://github.com/line/armeria/blob/15e228889d0bbace4c7ee08836831023f46d4aa0/it/spring/boot2-tomcat9/src/main/java/com/linecorp/armeria/spring/tomcat/demo/SpringTomcatApplication.java#L49

How about a more explicit `tomcatBasePath`?

Reply:
Yeah, let's extract it as a static constant `TOMCAT_BASE_PATH` and use it. 😄